When iasl is invoked with 'iasl -T ALL', and everything works correctly,
it still exits with a non-zero value.  Exit with zero if all is well.

Signed-off-by: Al Stone <ahs3@redhat.com>

Index: acpica-unix-20191018/source/compiler/asloptions.c
===================================================================
--- acpica-unix-20191018.orig/source/compiler/asloptions.c
+++ acpica-unix-20191018/source/compiler/asloptions.c
@@ -111,7 +111,7 @@ AslCommandLine (
         {
             exit (-1);
         }
-        exit (1);
+        exit (0);
     }
 
     /* Next parameter must be the input filename */
